modules/data-loading/pages/streaming-data-connector.adoc

```diff
@@ -0,0 +1,442 @@
+
+[tabs]
+====
+topic_person.json::
++
+--
+[,json]
+----
+{
+  "topic": "person-demo-104",
+  "partition_list": [
+    {
+      "start_offset": -2,
+      "partition": 0
+    }
+  ]
+}
+----
+--
+topic_friend.json::
++
+--
+[,json]
+----
+{
+  "topic": "friend-demo-104",
+  "partition_list": [
+    {
+      "start_offset": -2,
+      "partition": 0
+    }
+  ]
+}
+----
+--
+Loading job::
++
+--
+[.wrap,gsql]
+----
+CREATE LOADING JOB load_person FOR GRAPH social {
+    DEFINE FILENAME f1 = "$k1:/home/mydata/topic_person.json";
+    DEFINE FILENAME f2 = "$k1:/home/mydata/topic_friend.json";
+    LOAD f1 TO VERTEX person VALUES ($0, $0, $2, $1, $3) USING separator=",";
+    LOAD f2 TO EDGE friendship VALUES ($0, $1, $2)  USING separator=",";
+}
+----
+--
+====
+
+==== Run loading job
+Run the loading job created in the last step will load the streamed data into the graph.
+If you make changes to the topic-partition configuration file, you can overwrite the values for the filename variables with xref:gsql-ref:ddl-and-loading:running-a-loading-job.adoc#_run_loading_job[the `USING` clause].
+
+[,gsql]
+----
+GSQL > RUN LOADING JOB load_person
+----
+
+By default, loading jobs that use Kafka data sources run in streaming mode and do not stop until manually aborted.
+As data is streamed from the data source, the running loading job will continuously ingest the streamed data into the graph store.
+
+==== Stream data source updates
+
+To load updates to the data source after you've created the connectors, you need to delete the connector and recreate another connector.
+
+In the future, the streaming data connector will support automatically scanning for updates and stream data to TigerGraph.
+
+== Manege connectors
+
+After creating a connector, you can choose to delete it or pause it.
+You can also use `gadmin` commands to view status of all existing connectors.
+
+=== List connectors
+You can list all running connectors or view detailed configuration of a specific connector.
+
+To view a list of all connectors, run `gadmin connector list` from your bash terminal as the TigerGraph Linux user.
+
+To view configuration details of a specific connector, run `gadmin connector list <name_of_connector>` and replace `<name_of_connector>` with the name of the connector you want to inspect.
+
+=== Pause a connector
+Pausing a connector stops the connector from streaming data from the data source.
+The data that has been streamed to TigerGraph's internal Kafka can still be loaded into the graph store via a loading job.
+
+To pause a connector, run the below command and replace `<connector_name>` with the name of the connector:
+
+[,console]
+----
+$ gadmin connector pause <connector_name>
+----
+
+=== Resume a connector
+Resuming a connector resumes streaming for a paused connector.
+
+To resume a connector,  run the below command and replace `<connector_name>` with the name of the connector:
```

Comment:
To resume a connector, run the below command and replace `<connector_name>` with the name of the connector: